user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectSummaryComponent.java

```diff
@@ -218,6 +231,33 @@ private static List<? extends UserScopeStrategy> loadWriteScope(Section[] sectio
     return Arrays.stream(sections).map(WriteScopeStrategy::new).toList();
   }
 
+  private static SerializablePredicate<FundingEntry> incompletePredicate() {
+    return (FundingEntry entry) ->
+        !((entry.getLabel().isBlank() && !entry.getReferenceId().isBlank()) ||
+            (!entry.getLabel().isBlank() && entry.getReferenceId().isBlank()));
```

Comment:
```suggestion
    return (FundingEntry entry) ->
        (!entry.getLabel().isBlank() && !entry.getReferenceId().isBlank()) || (entry.getLabel().isBlank() && entry.getReferenceId().isBlank())
```
You can remove on level of complexity by inverting the operator. Instead of checking whether `not ( (left empty && right filled) or (left filled && right empty) )` you can check for `(left empty && right empty) or (left filled && right filled)`